### PR TITLE
Added LLNL wide language breakdown

### PR DESCRIPTION
**Pull request by aidan-epperly**
Adds similar breakdown of languages to the explore page as in the repo pages.

### CODE REVIEW

**Comment by aidan-epperly** [comment]
image

**Comment by IanLee1521** [comment]
That's looking good.

I assume if I moused over some of the smaller slices it would show what is there?

Maybe we need a sub-title in the pie for the units, e.g. "By # of Bytes of Code" or whatever the right answer is?

**Comment by LRWeber** [comment]
The byte count may be more meaningful to look at if it were written in familiar human readable terms (e.g. megabytes and gigabytes, as appropriate to the individual number).

Alternatively, or additionally, showing a percentage might be useful.


Is there a particular method to the data sorting? If not, ordering the slices by size like many of the other pie graphs could improve readability. It would also be better suited to the gradient based coloration (as seen in the Commit Breakdown pie graph).

File: js/explore/pie_languageExplore.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+            })
+        }
+    }
+
+    // Turn json obj into desired working data
+    function reformatData(obj) {
+        var languageData = {};
+        var repoData = {};
+
+        // Processs repo breakdown by language
```
Comment (LRWeber) [inline js/explore/pie_languageExplore.js:199]: Minor comment typo
```suggestion
        // Process repo breakdown by language
```

**Comment by IanLee1521** [comment]
> The byte count may be more meaningful to look at if it were written in familiar human readable terms (e.g. megabytes and gigabytes, as appropriate to the individual number).

Nod, I wasn't seeing any actual numbers so I was trying to get more at how the proportions where being done... is it bytes (I think it is)? Or lines of code? Or number of files? Or some other metric?

**Comment by LRWeber** [comment]
> Nod, I wasn't seeing any actual numbers so I was trying to get more at how the proportions were being done... is it bytes (I think it is)? Or lines of code? Or number of files? Or some other metric?

Currently bytes. It shows up on hover. I agree that it would help to have some indication in the always-visible labels.
image

**Comment by aidan-epperly** [comment]
> Is there a particular method to the data sorting? If not, ordering the slices by size like many of the other pie graphs could improve readability. It would also be better suited to the gradient based coloration (as seen in the Commit Breakdown pie graph).

I found that sorting, at least at the draw level, breaks the animation for the transition. I can sort the input data though.

> Maybe we need a sub-title in the pie for the units, e.g. "By # of Bytes of Code" or whatever the right answer is?

image

> The byte count may be more meaningful to look at if it were written in familiar human readable terms (e.g. megabytes and gigabytes, as appropriate to the individual number).

Sure, I can do that.

**Review by LRWeber — APPROVED** [review]

**Pull request merged**